src/components/QuizPage/QuizPage.tsx

```diff
@@ -1,20 +1,47 @@
-import React from 'react';
-import './Quiz.css';
+import React, {useEffect, useState} from 'react';
+import Form from '../Form/Form'
 
-const Quiz = () => {
+interface IState{
+  questions:Array<string> | string
+  currentCountry: {
+    name: string
+  } | undefined
+}
+
+const QuizPage: React.FC<IState> = ({questions, currentCountry}) => {
+  const [quizQuestions, setQuiz] = useState<IState['questions']>()
+  const [country, setCountry] = useState<IState['currentCountry']>()
+
+   // ********* Randomizer ********//
+  const getRandomElement = () => {
```

Comment:
Great job randomizing the quiz questions! This will make the game more fun for the user :)